docs/man_pages/index.md

```diff
@@ -11,7 +11,9 @@ Command | Description
 -------|----------
 [help `<Command>`](general/help.html) | Shows additional information about the commands in this list in the browser.
 [autocomplete](general/autocomplete.html) | Configures your current command-line completion settings.
-[feature-usage-tracking](general/feature-usage-tracking.html) | Configures anonymous feature usage tracking.
+[usage-reporting](usage-reporting.html) | Configures anonymous usage reporting for the NativeScript CLI.
+[error-reporting](error-reporting.html) | Configures anonymous error reporting for the NativeScript CLI.
+[doctor](doctor.html) | Checks your system for configuration problems which might prevent the NativeScript CLI from working properly.
```

Comment:
This will probably be an issue for the merge as I have just merged my changes for the doctor command.